### ISSUE TITLE
hope doc site improve user experience

### ISSUE THREAD

**Issue by yunyu950908** [nestjs/nest#1605]
<pre><code>
[ ] Regression 
[ ] Bug report
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

- hope https://docs.nestjs.com/ provide global (or keywords) searching ability
- see https://eggjs.org

**Comment by Sharlaan** [comment]
Yes, the official documentation is really missing some global search bar.
